packages/shared/src/components/fields/SearchField.tsx

```diff
@@ -102,31 +103,33 @@ export const SearchField = forwardRef(function SearchField(
       data-testid="searchField"
       ref={ref}
     >
-      {isSecondary && hasInput ? (
-        <Button
-          type="button"
-          className="mr-2 btn-tertiary"
-          buttonSize="xsmall"
-          title="Clear query"
-          onClick={onClearClick}
-          icon={
-            <CloseIcon className="text-lg icon group-hover:text-theme-label-primary" />
-          }
-          disabled={!hasInput}
-        />
-      ) : (
-        <SearchIcon
-          secondary={focused}
-          className="mr-2 text-2xl icon"
-          style={{
-            color:
-              focused || hasInput
-                ? 'var(--theme-label-primary)'
-                : 'var(--field-placeholder-color)',
-          }}
-        />
-      )}
+      {!!showIcon &&
```

Comment:
The `showIcon` should not be needed anymore, I think it was forgotten to be removed but as part of being a secondary type (in the search field guideline), icon should be displayed.

Reply:
New design does not have an icon
https://www.sketch.com/s/2867ec63-0966-4647-b1d7-b6fe696ba251/a/m1913YJ